### PR TITLE
update command

### PR DESCRIPTION
**Pull request by halababva**

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -344,7 +344,16 @@ RegisterNetEvent('ox_doorlock:breakLockpick', function()
...
 end)
 
-
-lib.addCommand(Config.CommandPrincipal, 'doorlock', function(source, args)
+lib.addCommand('doorlock', {
+    help = locale('create_modify_lock'),
+    params = {
+        {
+            name = 'closest',
+            help = 'Use for closest doors.',
```
Comment (thelindat) [inline server/main.lua:352]: 'Open UI to the closest door' or something?

**Comment by thelindat** [comment]
This requires bumping the dependency check for ox_lib to 3.0.0

**Comment by Manason** [comment]
> This requires bumping the dependency check for ox_lib to 3.0.0

resource still uses getCore right now. This PR is blocked

**Comment by thelindat** [comment]
> > This requires bumping the dependency check for ox_lib to 3.0.0
> 
> resource still uses getCore right now. This PR is blocked

Uhhhh getCore should be completely gone?

**Comment by cartiqo** [comment]
Does this work?

**Comment by Manason** [comment]
> > > This requires bumping the dependency check for ox_lib to 3.0.0
> > 
> > 
> > resource still uses getCore right now. This PR is blocked
> 
> Uhhhh getCore should be completely gone?

Right. My mistake. Screwed up my commit history in a fork and it was hanging around for me.

**Pull request closed without merging**